crypto/pem/pem_pkey.c

```diff
@@ -108,6 +108,11 @@ int PEM_write_bio_PrivateKey_traditional(BIO *bp, EVP_PKEY *x,
                                          pem_password_cb *cb, void *u)
 {
     char pem_str[80];
+
+    if (x->ameth == NULL || x->ameth->old_priv_encode == NULL) {
+        ERR_raise(ERR_LIB_PEM, PEM_R_UNSUPPORTED_PUBLIC_KEY_TYPE);
```

Comment:
Heh, should have looked at the Travis first - of course ERR_raise is 3.0 function.

Reply:
Oh, of course. Will fix

Reply:
Done